openid-4-verifiable-presentations-1_0.md

```diff
@@ -794,7 +794,11 @@ information disclosure to influence how they order these options. For example, a
 proof of age request should prioritize requesting an attribute like
 `age_over_18` over an attribute like `birth_date`. The `claim_sets` syntax is
 not intended to define options the user can choose from, see (#dcql_query_ui) for
-more information.
+more information. The Wallet is recommended to return the first option it can satisfy
+since that is the preferred option from the Verifier. However there can be reasons to
+deviate. Examples of such reasons are; scenarios where the Verifier did not order 
```

Comment:
Is that semicolon correct here? Not a native speaker, but this sentence just looks weird to me this way.

Reply:
just make it a bullet point list, now?